component/main.jsonnet

```
@@ -5,6 +5,67 @@ local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.machineset_egress_cidr_operator;
 
+local namespace = kube.Namespace(params.namespace) {
+  metadata+: {
+    labels+: params.labels,
+  },
+};
+
+local serviceAccount = kube.ServiceAccount('machineset-egress-cidr-operator') {
+  metadata+: {
+    labels+: params.labels,
+  },
+};
+
+local clusterRole = kube.ClusterRole('machineset-egress-cidr-operator') {
+  metadata+: {
+    labels+: params.labels,
+  },
+  rules: params.clusterRole_rules,
```

Search & replace collateral damage
```suggestion
  rules: params.cluster_role_rules,
```